### DISCUSSION TITLE
Ερώτηση για την άσκηση, συμμετοχικό περιεχόμενο 2Α.

### DISCUSSION THREAD

**Discussion by akimo13** [upatras-hci/iv#70] [Q&A] [answered]
Καλησπέρα @epidrome

Σύμφωνα με την εκφώνηση της άσκησης 2Α, καλούμαστε να προσθέσουμε στο βιβλίο δύο νέα διαδραστικά παραδείγματα, ο κώδικάς των οποίων θα παρουσιάζεται σε κάποιο social development environment, ενδεικτικά σαν το [Codepen](https://codepen.io/).

Θα ήθελα λοιπόν να ρωτήσω:

1. Τον κώδικα για τα παραδείγματα χρειάζεται να τον αναπτύξουμε εξ'ολοκλήρου εμείς;
2. Μπορούμε απλά να μοιραστούμε κάποιο έτοιμο παράδειγμα που θα βρούμε και προσφέρει διαδραστικότητα σύμφωνα με τα κριτήρια που αναφέρονται στις οδηγίες;
3. Τα παραδείγματα θα αφορούν κάποιο, τροποποιημένο από εμάς, έτοιμο κώδικα που έχουμε βρει και πληροί τα κριτήρια της άσκησης;

Ευχαριστώ

**Comment by e-liouta** [comment, marked as answer]
Εγώ πάντως αυτό που έκανα είναι ότι σκέφτηκα τι θέλω να κάνω έριξα μια ματιά στο τι υπάρχει ήδη και έφτιαξα κάποια δικά μου συνδυασμό ήδη υπαρχόντων και δικών μου προσθηκών.

**Reply by epidrome** [reply]
σχεδόν κανένα από αυτά στην σελίδα δεν είναι από το μηδέν, 

ένα σημαντικό μέρος της άσκησης είναι ο εντοπισμός και η μελέτη σχετικού υλικού

εξίσου σημαντικό είναι να γίνουν αλλαγές-προσθήκες και όχι απλή αντιγραφή

**Comment by akimo13** [comment]
Ωραία και δημιουργική ιδέα @e-liouta !
